applications/luci-app-aria2/root/usr/share/rpcd/acl.d/luci-app-aria2.json

```diff
@@ -2,6 +2,18 @@
 	"luci-app-aria2": {
 		"description": "Grant UCI access for luci-app-aria2",
 		"read": {
+			"ubus": {
+				"service": [ "list" ]
+			},
+			"file": {
+				"/etc/group": [ "read" ],
+				"/etc/passwd": [ "read" ],
+				"/bin/cat": [ "exec" ],
+				"/bin/grep": [ "exec" ],
+				"/sbin/logread": [ "exec" ],
+				"/usr/bin/aria2c": [ "exec" ],
+				"/usr/bin/tail": [ "exec" ]
+			},
```

Comment:
Does we need this commands. I have not seen the usage in your code?

- "/etc/group": [ "read" ]
- "/etc/passwd": [ "read" ]

Reply:
They’re needed if the UserSelect or GroupSelect ui widgets are used

Reply:
Only `UserSelect` is used and I have removed the read access to `/etc/group`.